project/project-info.conf

```diff
@@ -122,4 +122,14 @@ project-info {
       }
     ]
   }
+  rolling-update-kubernetes: ${project-info.shared-info} {
+    title: "Akka Rolling Update Kubernetes"
+    levels: [
+      {
+        readiness: Supported
+        since: "2023-03-01" #FIXME
+        since-version: "1.2.0" #FIXME
```

Comment:
Still not sure what to do here...

Reply:
I suppose the new version will be 1.2.1? or 1.3.0?
And support date probably from april? 2023-04-01 ?

Reply:
Yes, it will be a minor bump, 1.3.0, because we changed some dependencies with minor bump (Akka HTTP for Scala 3).

2023-04-01 sounds good, not important to set that exactly right to the release date